graph/src/data/query/error.rs

```diff
@@ -14,15 +19,25 @@ pub enum QueryExecutionError {
     NoRootQueryObjectType,
     NoRootSubscriptionObjectType,
     ResolveEntityError(Pos, String),
+    ResolveListError,
     NonNullError(Pos, String),
     ListValueError(Pos, String),
     NamedTypeError(String),
+    ObjectFieldError,
     AbstractTypeError(String),
     InvalidArgumentError(Pos, String, q::Value),
     MissingArgumentError(Pos, String),
+    StoreQueryError(String),
+    FilterNotSupportedError(String, String),
     UnknownField(Pos, String, String),
     EmptyQuery,
     MultipleSubscriptionFields,
+    SubgraphParseError(String),
+    BuildRangeTypeError,
+    BuildFilterError,
+    EntityAttributeError,
```

Comment:
Let's make this report the entity type and attribute name like so:
```rust
EntityAttributeError(String, String)
```
and use this for the error message:
```rust
"Entity '{}' has no attribute '{}'"
```